### DISCUSSION TITLE
robots.txt and bad search engine results

### DISCUSSION THREAD

**Discussion by nzyeehaa** [joomla/joomla-cms#38129] [2. Cold Features]
new J414 site indexed in search engines shows 
- website.com/component/users
- with title home/
- link website.com/component/users?view=reset

robots.txt is active and includes Disallow: /component**s**/

suggest to **also** add
`Disallow: /component/` with no "s"

**Comment by brianteeman** [comment]
https://developers.google.com/search/docs/advanced/robots/intro

**Comment by nzyeehaa** [comment]
with login and logout pages already set with noindex,nofollow
and log out page restricted to registered
and no links to login page
and login page protected by cookies / htaccess / 404

for search engines to find `users?view=reset `- it needs a link to follow, and link must be in the core

is the best way to treat a joomla website to noindex, nofollow the entire site, then index,follow the category/pages you want listed in search engines ??

**Comment by Hackwar** [comment]
Sorry, but this is your subjective opinion on this and is best solved by you adapting your robots.txt to what you need. We wont change the robots.txt to rejecet /component/.
